src/components/chip/dropdown-chip-single-select/index.tsx

```diff
@@ -66,6 +66,9 @@ export const DropdownChipSingleSelect = forwardRef<
                   labelTag={labelTag}
                   ref={ref}
                   dropdown
+                  dropdownItemSelected={
+                    selectedItem.value !== defaultOption.value
+                  }
                   isDropdownOpen={open}
                   disabled={disabled}
                   {...rest}
```

Comment:
@shafin-deriv, if someone passes in a prop `dropdownItemSelected`, it'll overwrite yours, is that what's intended?
Maybe you could move the  `{...rest}` to the top, before any other prop.

Reply:
Done ✅